inttest/network-conformance/network_test.go

```diff
@@ -100,12 +102,37 @@ func (s *networkSuite) TestK0sGetsUp() {
 	status, err := client.GetStatus(&sc.StatusConfig{Namespace: "sonobuoy"})
 	s.Require().NoError(err)
 
+	s.T().Log("retrieving results")
+	r, ec, err := client.RetrieveResults(&sc.RetrieveConfig{
+		Namespace: "sonobuoy",
+		Path:      "/tmp/sonobuoy",
+	})
+	s.Require().NoError(err)
+	s.Require().NoError(retrieveResults(r, ec))
+
 	s.T().Log("sonobuoy test status: ", status)
 	s.Require().Equal("complete", status.Status)
 	s.Require().Len(status.Plugins, 1)
 	s.Require().Equal("passed", status.Plugins[0].ResultStatus)
 }
 
+func retrieveResults(r io.Reader, ec <-chan error) error {
+	eg := &errgroup.Group{}
+	eg.Go(func() error { return <-ec })
+	eg.Go(func() error {
+		filesCreated, err := sc.UntarAll(r, os.TempDir(), "")
```

Comment:
We could also just write the tar file as-is to disk, so we have to deal with a single file only, instead of a bunch of them.